irohad/torii/impl/command_service_transport_grpc.cpp

```diff
@@ -150,7 +162,20 @@ namespace iroha {
             });
       }
 
-      return grpc::Status::OK;
+      auto status = grpc::Status::OK;
+      if (not unprocessed_batches.empty()) {
+        std::stringstream message;
+        message << "The batches with the following reduced hashes were not "
```

Comment:
In an offline discussion we decided to make a generic error message without hashes.